src/app/organization/holidays/view-holidays/view-holidays.component.html

```diff
@@ -0,0 +1,65 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" (click)="activateHoliday()" *ngIf="holidayData.status.value !== 'Active'">
```

Comment:
![Screenshot from 2020-05-22 13-39-54](https://user-images.githubusercontent.com/54709463/82646905-fae79400-9c32-11ea-8776-5f3c10fdda8f.png)
Make activate button look like this (refer self service section)
Update screenshot after changes